templates/konvoy-ui.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: konvoy-ui
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: konvoy-ui
+  annotations:
+    appversion.kubeaddons.mesosphere.io/konvoy-ui: "1.47.2"
+    endpoint.kubeaddons.mesosphere.io/konvoy-ui: /ops/portal/
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  cloudProvider:
+    - name: aws
+      enabled: true
+    - name: docker
+      enabled: true
+    - name: none
+      enabled: true
+  chartReference:
+    chart: kommander
+    repo: https://mesosphere.github.io/charts/stable
+    version: 0.1.10
+    values: |
+      image:
+        tag: 1.47.2
```

Comment:
@s12chung I added the latest docker image tag, instead of `relpath`

Reply:
🤞

Reply:
it's the default value, we can remove this: https://github.com/mesosphere/charts/blob/dev/stable/kommander/values.yaml#L3

forgot to remove it when I pushed up. my bad.